Git-and-GitHub.txt

```diff
@@ -0,0 +1,31 @@
+--> to check the current user details
+
+	$ git config --list
+
+
+
+
+
+
+
+===> to change account in github desktop
+	======================================
+
```

Comment:
don'tgive this much space here it not look nnice

Reply:
ok, i will change it